pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -35,6 +35,8 @@
     <ExcludeFromClosure Include="System.Net" />
     <ExcludeFromClosure Include="System.Transactions" />
 
+    <ExcludeFromDuplicateTypes Include="System.Private.Reflection.Metadata.Ecma335" />
```

Comment:
I don't know how to make things in an assembly internal without forking the code of S.R.Metadata and writing internal everywhere (Or a truly horrific quantity of # if statements). The entire purpose of this assembly is to avoid said forking and # if operations, so I don't know how to do that.